service/local-dev/run_server.sh

```diff
@@ -1,13 +1,34 @@
 #!/bin/bash
 
-export TANAGRA_DATABASE_NAME=tanagra_db
-export TANAGRA_DB_INITIALIZE_ON_START=true
-export TANAGRA_DB_USERNAME=dbuser
-export TANAGRA_DB_PASSWORD=dbpwd
-
-export TANAGRA_UNDERLAY_FILES=broad/aou_synthetic/expanded/aou_synthetic.json,broad/cms_synpuf/expanded/cms_synpuf.json
-export TANAGRA_FEATURE_ARTIFACT_STORAGE_ENABLED=true
-export TANAGRA_AUTH_IAP_GKE_JWT=false
-export TANAGRA_AUTH_BEARER_TOKEN=true
+setupEnvVars() {
+  disableAuthChecks=$1 # 0=false, 1=true
```

Comment:
opt Since this file is only for local development, should we take out `disableAuthChecks` and just do lines 16-17?

Reply:
This is only for local UI development. For local backend development, i.e. where we're testing with the Swagger page, we still want authentication enabled. I added the bearer token authentication method specifically to support backend testing on a local env (i.e. not behind IAP).

Reply:
Updated `CONTRIBUTING.md`